pkg/services/sqlstore/migrations/phlare_to_grafana_pyroscope_rename.go

```diff
@@ -0,0 +1,12 @@
+package migrations
+
+import "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+
+func changePhlareIdToGrafanaPyroscope(mg *migrator.Migrator) {
+	mg.AddMigration("update datasource type",
```

Comment:
```suggestion
	mg.AddMigration("migrate phlare data_source to pyroscope",
```
the name here is used as a key, and must be unique across all migrations, so lets pick something more specific